### PR TITLE
Misc cleanups

### PR DESCRIPTION
**Pull request by cboesch-gpsw**

### CODE REVIEW

File: libnodegl/backends/gl/texture_gl.h [hunk 1]
```diff
@@ -36,6 +36,10 @@ struct texture_gl {
     GLint format;
     GLint internal_format;
     GLenum format_type;
+
```
Comment (mbouron-gpsw) [inline libnodegl/backends/gl/texture_gl.h:39]: Can you remove this extra line ?

**Review by mbouron-gpsw — APPROVED** [review]
Except from my comment, LGTM.

File: libnodegl/backends/gl/texture_gl.h (hunk 1 above)
Comment (cboesch-gpsw) [reply libnodegl/backends/gl/texture_gl.h]: Yup done

**Comment by mbouron-gpsw** [comment]
Do you mind scoping only the wrapped field to texture_gl in your last commit ? I will provide a commit to make use of external_storage / bytes_per_pixel in texture_vk if that's ok with you ?

**Comment by cboesch-gpsw** [comment]
> Do you mind scoping only the wrapped field to texture_gl in your last commit ? I will provide a commit to make use of external_storage / bytes_per_pixel in texture_vk if that's ok with you ?

Is this going to be set in the common path or every backend is going to have the responsibility to keep these fields consistent?

**Comment by mbouron-gpsw** [comment]
> Is this going to be set in the common path or every backend is going to have the responsibility to keep these fields consistent?

I guess yes but after a third thought, you can keep your commit as is. Sorry for the noise.

**Pull request merged**